sdk/keyvault/keyvault-keys/test/public/localCryptography.spec.ts

```diff
@@ -222,6 +217,7 @@ describe("Local cryptography public tests", () => {
           digest,
           signature.result
         );
+        // TODO: assert.ok(verifyResult.result) I think this test is broken
```

Comment:
What I mean by that is that `verifyResult` is a non-empty object and is always truthy. What we want to assert is the roundtrip which unfortunately currently isn't returning true... I'll look into this as well before merging.